C++/2. Variables.md

```diff
@@ -2,59 +2,78 @@
 
 ## Start
 
-Variables are containers which hold a specific something. They are words which contain something in them, which you can call or take by just typing out the word. Variables in C++ are created by first typing out their type. Each variable has a type, and there are a few types of variables:
+Variables are containers which can hold values of a specific type. They are words which contain something in them, which you can call or take by just typing out the word. Variables in C++ are created by first typing out their type. Each variable has a type, and there are an infinite number of types that you can create. The language provides a few primitives types, which can be used to build larger, or more complicated types, here are a few:
 
+- `bool`
+- `char`
+- `short`
+- `unsigned`
 - `int`
+- `long`
 - `float`
+- `long long`
 - `double`
-- `std::string`
-- `char`
-- `bool`
```

Comment:
Hey, why did you remove std::string. Also isn't it `long int` and `long long int`

Reply:
It was to fit in with the ideas mentioned right before it, that C++ is low-level, and that these are the primitive types in C++.

I wasn't sure if I wanted to explicitly use int or not, but `long x = 0` is totally valid.

I'll add 'em in soon!

Reply:
Oh yeah, and I'll add `std::string` back in, if you want.